src/AdminDashboard/Fleets/FleetsFilesUpload.js

```diff
@@ -15,7 +15,7 @@ export default class FleetsFilesUpload extends Component {
         this.setState({
           files
         }, () => {
-        //   console.log(this.state.files)
+            //console.log(this.state.files)
```

Comment:
why do you leave these comments?

Reply:
They were in the code, I thought it was important.
Should I delete all comments like those?

Reply:
@Gianfranco97 could you check my changes in the MessageEmpty and Loading component? To stop adding unnecessary props ? It's a detail